azure/src/main/scala/quasar/destination/avalanche/azure/AvalancheAzureDestinationModule.scala

```diff
@@ -46,18 +51,18 @@ import eu.timepit.refined.auto._
 
 import doobie.hikari.HikariTransactor
 
-object AvalancheDestinationModule extends DestinationModule {
+object AvalancheAzureDestinationModule extends DestinationModule {
   val IngresDriverFqcn = "com.ingres.jdbc.IngresDriver"
   // Avalanche closes the connection after 4 minutes so we set a connection lifetime of 3 minutes.
   val MaxLifetime = 3.minutes
   val Redacted = "<REDACTED>"
   val PoolSize: Int = 10
 
   def destinationType: DestinationType =
-    DestinationType("avalanche", 1L)
+    DestinationType("avalanche-azure", 1L)
```

Comment:
This needs to be coordinated with the frontend to ensure the UI is appropriately updated.